source/scriptHandler.py

```diff
@@ -54,8 +54,11 @@ def _getObjScript(obj, gesture, globalMapScripts):
 			except AttributeError:
 				pass
 
-	# Search the object itself for in-built bindings.
-	return obj.getScript(gesture)
+	try:
+		# Search the object itself for in-built bindings.
+		return obj.getScript(gesture)
+	except:  # noqa: E722 do not use bare 'except'
+		log.exception()

```

Comment:
May be an explanatory string here as well?

Reply:
Done:  ca1a6fb